DESIGN.md

```
@@ -0,0 +1,1311 @@
+  [P] Pack Environment Files
+> [U] Unpack Archive              <- Selected (teal chevron)
+  [L] List Archive Contents
+  [S] Status
+  [●] Settings
+  [?] Help
+```
+
+**Styling:**
+- Unselected: No prefix, normal text color
+- Selected: Teal `>` prefix, text remains normal (no background highlight)
+- Two-space indent for unselected items to align with `> ` prefix
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)